projects/core/src/configurator/commons/connectors/configurator-commons.connector.ts

```diff
@@ -54,4 +54,11 @@ export class ConfiguratorCommonsConnector {
   ): Observable<Configurator.Configuration> {
     return this.adapter.readPriceSummary(configuration);
   }
+
+  getConfigurationOverview(
+    configId: string,
+    owner: Configurator.Owner
+  ): Observable<Configurator.Configuration> {
+    return this.adapter.getConfigurationOverview(configId, owner);
```

Comment:
Should return Observable<Configurator.Overview>.
Still the effect could instantiate  GetConfigurationOverviewSuccess(configuration) with ownerKey and overview which is then in the reducer added to the store

Reply:
Done.